hummingbot/client/ui/hummingbot_cli.py

```diff
@@ -70,6 +69,8 @@ def __init__(self,
         loop.create_task(start_trade_monitor(self.trade_monitor))
 
     async def run(self):
+        self.app = Application(layout=self.layout, full_screen=True, key_bindings=self.bindings, style=load_style(),
+                               mouse_support=True, clipboard=PyperclipClipboard())
```

Comment:
Is there any particular reason for this?

Reply:
This is to avoid having to mock the `Application` startup when using the CLI in tests